examples/session.rs

```diff
@@ -0,0 +1,522 @@
+//! Simple WalletConnect v2 client to demonstrate Sign API functionality.
```

Comment:
Can we move this example to `sign_api` folder?

Reply:
Done.